### ISSUE TITLE
Bug: The container kitchenowl-web can't start

### ISSUE THREAD

**Issue by xblr** [TomBursch/kitchenowl#105]
### Is this urgent?

Yes

### What parts are affected

Frontend

### What are the Server/Client versions of KitchenOwl

https://hub.docker.com/layers/tombursch/kitchenowl-web/latest/images/sha256-edcb9d526c76ef891d8ac157af3e4aca12fc750efc2c91767b2f393fc85d788d

### What's the problem 🤔

The docker container can't start

### Share your logs

```log
/docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration,
/docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/,
/docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh,
10-listen-on-ipv6-by-default.sh: info: ipv6 not available,
/docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh,
20-envsubst-on-templates.sh: Running envsubst on /etc/nginx/templates/default.conf.template to /etc/nginx/conf.d/default.conf,
/docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh,
/docker-entrypoint.sh: Configuration complete; ready for start up,
2022/12/07 15:50:52 [emerg] 1#1: socket() [::]:80 failed (97: Address family not supported by protocol),
nginx: [emerg] socket() [::]:80 failed (97: Address family not supported by protocol)
```

### Share your configuration

```yml
version: "3"
services:
  front:
    image: tombursch/kitchenowl-web:latest
    # environment:
    #   - BACK_URL=back:5000 # Optional should not be changed unless you know what youre doing
    ports:
      - "3003:80"
    depends_on:
      - back
    networks:
      - default
  back:
    image: tombursch/kitchenowl:latest
    restart: unless-stopped
    # ports: # Optional
    #   - "80:80" # http protocol
    #   - "5000:5000" # uwsgi protocol
    networks:
      - default
    environment:
      - JWT_SECRET_KEY='xxxxxxxxxxxxxx'
      # - FRONT_URL=http://localhost # Optional should not be changed unless you know what youre doing
    volumes:
    - kitchenowl_data:/data

volumes:
  kitchenowl_data:

networks:
  default:
```

**Comment by xblr** [comment]
I have the ipv6 disable in my machine, I don't know if that is the problem
Thanks

**Comment by TomBursch** [comment]
> I have the ipv6 disable in my machine, I don't know if that is the problem

It has definitely something to do with this. [NGINX](https://github.com/TomBursch/kitchenowl/blob/main/default.conf.template) listens on both stacks. I'll look for a solution.

**Comment by MinehubDE** [comment]
The pull request #110 should fix this issue

**Comment by TomBursch** [comment]
Fixed with #110
